corehq/apps/sms/migrations/0050_sms_email_date_modified.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.24 on 2021-06-29 13:16
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sms', '0049_auto_enable_turnio_ff'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='email',
+            name='date_modified',
+            field=models.DateTimeField(auto_now=True, db_index=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='sms',
+            name='date_modified',
+            field=models.DateTimeField(auto_now=True, db_index=True, null=True),
```

Comment:
Is there any advantage in updating these with `date` column or that won't be correct reflection of when the record was last modified?

Reply:
you mean using date instead of datetime? We need the time part otherwise you won't be able to get changes at the sub-day level.

Reply:
There is a column "date" which is a datetime, and is I believe is the timestamp for object creation.
Just thinking if we can and should be populating the new column for existing records, with any other timestamp that is available.
The value for the new column would be NULL for existing records, that won't conflict with how we expect them to order in the API right?

Reply:
That's correct, existing dates will be NULL which is fine. The API will use `GREATEST(event.date, sms.date_modified, email.date_modified)` which ignores null values.

Reply:
Great, thank you